<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>EVO</title>
<link href="EVO.css" rel="stylesheet" type="text/css" />
<script type="text/javascript">
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
</script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-39398660-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<style type="text/css">
<!--
a:link {
	text-decoration: none;
	color: #999;
}
a:visited {
	text-decoration: none;
	color: #999;
}
a:hover {
	text-decoration: none;
	color: #0CC;
}
a:active {
	text-decoration: none;
	color: #0CC;
	text-align: center;
}
a {
}
-->
</style>
<body>
<div id="container">
  <div id="main_image">
  	<div id="menu">
    <p><a href="index.html">about</a></p>
    <p><a href="projects.html">project list</a></p>
    <p><a href="contact.html">contact</a></p>
    <p><a href="http://www.drustvo-evo.hr">EVO</a></p>
    <p><a href="links.html"> links</a></p>
  	</div>
    <div id="text_box">
      <p><a href="index.html"><img src="images/EVO_internship_portal_logo.png" width="200" height="114" /></a></p>
      <p>&nbsp;</p>
      <h2><strong>List of projects/laboratories offering internships</strong></h2>
      <ul>
        <li><strong><a href="#P1">Microfluidic solutions to chemical and biological problems</a></strong></li>
        <li><strong><a href="#P2">Telomeres and genome integrity</a></strong></li>
        <li><strong><a href="#P3">Novel photoswitchable organocatalysts based on molecular motors</a></strong></li>
        <li><strong><a href="#P4">Synthesis and characterization of E-selectin binding peptides for the development of multimodal imaging probes</a></strong></li>
        <li><span lang="EN-GB" xml:lang="EN-GB"><strong><a href="#P9">Development of metallopeptides as catalysts for greener chemical transformations</a></strong></span></li>
        <li><strong><a href="#P5">Improving robustness of host cells for external environments of varying and changing osmolalilites</a></strong></li>
        <li><strong><a href="#P6">What happens to the anomalies of super-cooled salty water?</a></strong></li>
        <li><strong><a href="#P7">Developing new simulation methods for the modelling of supramolecular Interactions</a></strong></li>
        <li><strong><a href="#P8">A computational genomics approach to enhancer-promoter interactions</a></strong></li>
		<li><strong><a href="#P10">Genetic code expansion and reprogramming</a></strong></li>
		<li><strong><a href="#P11">Modifying G protein-coupled receptors for crystallographic studies</a></strong></li>
      </ul>
      <p>&nbsp;</p>
      <hr />
      <p><strong><a name="P1" id="P1"></a><h3>Microfluidic solutions to chemical and biological problems - occupied</h3></strong><br />
</p>
      <p><strong>A short description of the project </strong><br />
        The deMello group specialises in microfluidic technology and nanoscale science. Research performed in the group covers areas such as the development of microfluidic devices for high-throughput biological and chemical analysis, nanofluidic reaction systems for chemical synthesis, ultra-sensitive optical detection techniques, novel methods for nanoparticle synthesis, the exploitation of semiconducting materials in diagnostic applications, the development of intelligent microfluidics and the processing of living organisms. If you are interested in doing a project with the deMello group, please take a look at our currently available projects (<a href="http://www.demellogroup.ethz.ch/en/mastersprojects.php"><strong>http://www.demellogroup.ethz.ch/en/mastersprojects.php</strong></a>).</p>
      <p><strong>Minimum time (e.g. 3-6 months) </strong><br />
        Open.</p>
      <p><strong>Possible start </strong><br />
        Open.      </p>
      <p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
        CV and a short motivational letter.</p>
      <p><strong>Contact information </strong></p>
      <p><img src="images/K_Elvira.png" width="84" height="122" hspace="9" align="left" />Dr Katherine Elvira<br />
        Senior Scientist<br />
        Institute for Chemical and Bioengineering<br />
        Department of Chemistry and Applied Biosciences<br />
        ETH&nbsp;Zürich</p>
      <p>Email: katherine.elvira AT chem.ethz.ch </p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <p>&nbsp;</p>
      <hr />
      
      <p><strong><a name="P2" id="P2"></a><h3>Telomeres and Genome Integrity - occupied</h3></strong><br />
</p>
      <p><strong>A short description of the project</strong> <br />
      The European Research Institute for the Biology of Ageing (ERIBA) is a newly established research institute at the University Medical Center Groningen (UMCG) in the Netherlands. The central theme at ERIBA is the mechanisms of ageing. We aim to identify physiological and molecular changes that can explain the loss of cells and functional decline of cells and tissues with age to achieve and promote healthy ageing. To do so, we make use of state of the art technology and a variety of model organisms. ERIBA is housed in a brand new building and is part of the UMCG campus in central Groningen, a small (200,000 inhabitants) but vibrant city (with over 25,000 students) in the north of the Netherlands.      </p>
<p>The Laboratory of Telomeres and Genome Integrity at ERIBA, headed by Dr. Michael Chang, aims to characterize mechanisms of telomere maintenance, with the ultimate goal of elucidating the role of telomeres in cancer progression and replicative ageing.</p>
      <p>Telomeres, the physical ends of eukaryotic chromosomes, help distinguish natural chromosome ends from DNA breaks in need of repair. Dysfunctional telomeres result in DNA damage checkpoint activation and cell cycle arrest. Telomeres progressively shorten due to incomplete DNA replication and nucleolytic degradation. When telomeres are critically shortened, cells can no longer divide, reaching a state known as replicative senescence. Shortening is counteracted by telomerase, the specialized reverse transcriptase that elongates telomeres. However, most human somatic cells express little or no telomerase, so telomeres erode with each passing cell division, ultimately leading to replicative senescence. This process has been proposed to be one determinant of organismal ageing.</p>
      <p><strong>Techniques covered in the project </strong><br />
        Research in the Chang lab combines classical genetic and molecular biological approaches with high-throughput genomic methodologies using the budding yeast Saccharomyces cerevisiae as a model organism to study telomere length regulation.      </p>
      <p><strong>Minimum time (e.g. 3-6 months) </strong><br />
      Open.</p>
      <p><strong>Possible start </strong><br />
        Open.      </p>
      <p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
        CV and a motivational letter.</p>
      <p><strong>Contact information </strong></p>
      <img src="images/M_Chang.png" width="134" height="118" hspace="9" align="left" />For more information about Dr. Chang and ERIBA, see <a href="http://www.eriba.umcg.nl"><strong>http://www.eriba.umcg.nl</strong></a>. <br />
	  <p> m.chang AT umcg.nl </p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr />
<p><strong><a name="P3" id="P3"></a><h3>Novel photoswitchable organocatalysts based on molecular motors - occupied</h3></strong><br />
</p>
<p><strong>A short description of the project </strong></p>
<p><img src="images/Fig1.png" width="575" height="228" /><br />
  <strong>Figure 1.</strong> Rotary cycle of a molecular motor</p>
<p>Photoswitchable catalysis is a fast growing field in which photochemical processes are used to alter the result of chemically catalyzed reactions. Unique amongst photoswitchable molecules, light driven molecular motors are able to convert photochemical energy directly into rotary motion which is realized within a sequence of transformations (depicted in Figure 1). In special cases the energy barrier between those transformations is big enough to isolate the motor molecule in three different states (I, II and III on Figure 2), which makes it act as a ternary molecular switch. State I corresponds to the &quot;off&quot; state in which the &quot;rotor&quot; and &quot;stator&quot; part of the molecule point in opposite directions. States II and III correspond to two different &quot;on&quot; states in which the &quot;rotor&quot; and &quot;stator&quot; are pointing to the same side of the molecule. Steric hindrance causes a thermal energy barrier between states II and III which makes it possible to isolate both forms at room temperature.States II and III have the opposite sense of axial chirality - M or P. They are so called pseudoenantiomers.</p>
<p>&nbsp;</p>
<p><img src="images/Fig2.png" width="572" height="315" /></p>
<p><strong>Figure 2.</strong> Motor organocatalyst contains DMAP and thiourea as catalytically active parts and exists in 3 different states which. States II and III give different stereochemical outcome of Michael addition of 2-metoxythiophenol to cyclohexenone</p>
<p>In 2011 (Science, 2011, 331, 1429-1432) we have demonstrated that by using a molecular motor bearing two different components of an organocatalytic system on the upper (&quot;rotor&quot;) and lower (&quot;stator&quot;) halves respectively, it is possible to control the stereochemical outcome of a reaction by an external trigger (light or heat). In other words, while state I remains catalytically inactive, the pseudoenantiomeric states II and III catalyze the reaction providing opposite enantiomers of the product. This was the first reported example in which one catalyst was able to provide two different enantiomers of the product in a controlled fashion. The purpose of the project would be development of novel photoswitchable systems that could be used in organocatalysis.</p>
<p><strong>Techniques covered in the project </strong> <br />
  Organic synthesis&nbsp; under inert conditions, working in glove-box, HPLC analysis, UV/vis , CD and NMR</p>
<p><strong>Minimum time (e.g. 3-6 months) </strong><br />
  4 months</p>
<p><strong>Possible start </strong><br />
  From 13.1.2014. onwards</p>
<p><strong>Student requirements (e.g. GPA, courses taken, previous experience) </strong><br />
  Taken organic chemistry classes and practicums.</p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
  CV and a motivational letter.</p>
<p><strong>Contact information </strong></p>
<p><img src="images/M_Vlatkovic.png" width="98" height="181" hspace="9" align="left" />Matea Vlatkovic <br />
PhD student <br />
Faculty of Mathematics and Natural Sciences<br />
University of Groningen<br />
  m.vlatkovic AT rug.nl</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr />
<p><strong><a name="P4" id="P4"></a><h3>Synthesis and characterization of E-selectin binding peptides for the development of multimodal imaging probes - occupied</h3></strong> <br />
 </p>
<p><strong>A short description of the project </strong><br />
E-selectin is a cell adhesion molecule which is induced on the surface of endothelial cells in response to inflammatory cytokines. Consequently, elevated E-selectin expression has been reported in many types of chronic and acute inflammation diseases (e.g. atherosclerosis and derived cardiovascular diseases, and arthritis) as well as cancer. This different expression profile makes E-selectin an attractive approach to detect these diseases that account for the greatest number of deaths worldwide and are becoming a major global health burden particularly with the aging of the global population. Different noninvasive imaging probes have been developed and used for E-selectin detection. However, despite the benefits and potential of probes that combine multiple imaging modalities in a single platform, only few have been reported so far. Therefore, we intent to develop and evaluate noninvasive multimodal molecular imaging probes that will allow in vivo mapping of E-selectin expression by combined ultrasound (US), magnetic resonance (MR) and optical (florescence and NIR-fluorescence) imaging. </p>
 <p> Under the framework of this project, the candidate will: </p>
<ul>
  <li>synthesize and characterize different peptidic constructs that have been computationally designed to bind E-selectin. These designs were carried out in our laboratory using as a starting model a lead construct already tested both in vitro and in vivo <br />
  </li>
  <li>study their interaction with the receptor using different methodologies such as NMR (Saturation transfer difference and Diffusion-ordered techniques) and fluorescence spectroscopies </li>
</ul>
<p><strong>Techniques covered in the project </strong> <br />
  Solid phase and solution peptide synthesis methodologies <br />
  Mass spectrometry <br />
  Analytical and preparative HPLC <br />
  Spectroscopic techniques: Circular dichroism, fluorescence, NMR and UV-Vis </p>
<p><strong>Minimum time (e.g. 3-6 months) </strong> <br />
  3 months, preferably 6 months </p>
<p><strong>Possible start </strong> <br />
  To be determined with the candidate </p>
<p><strong>Student requirements (e.g. GPA, courses taken, previous experience) </strong> <br />
  Preferably chemistry/biochemistry background </p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong> <br />
  Please, send by email your detailed CV, motivation letter and the contact information of two referees. </p>
<p><strong>Contact information </strong></p>
<p><img src="images/O_Iranzo.png" width="121" height="118" hspace="9" align="left" />Olga Iranzo <br />
  Institut des Sciences Moléculaires de Marseille <br />
  ISM2/BiosCiences UMR CNRS 7313, case 341 <br />
  Aix-Marseille Université, Campus Scientifique de Saint Jérôme <br />
  Avenue Escadrille Normandie-Niemen <br />
  F-13397 Marseille CEDEX 20 <br />
  Phone number: 00 33 (0)4 91 28 27 00 <br />
  Fax number: 00 33 (0)4 91 28 84<br />
  olga.iranzo AT univ-amu.fr</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
  
  <hr />
  
<p><strong><a name="P9" id="P9"></a><h3>Development of metallopeptides as catalysts for greener chemical transformations - occupied</h3> </strong> <br />
   
<p><strong>A short description of the project  </strong>
  
<p>Amino acids containing side chain modifications that increase their metal ion binding affinities have applications in very diverse research fields, among them the development of new catalysts. The possibility of combining the reactivity of a transition metal center with the enantioselective inducing environment and water solubility properties of a peptidic scaffold becomes an attractive approach to develop new catalysts for chemical transformations. In this context, the aim of this project is to develop metallopeptides as catalysts for oxidative processes using green oxidants such as hydrogen peroxide (H2O2) and oxygen (O2). The use of H2O2 is particularly attractive because it is inexpensive and produces water as the only side product and O2 has been targeted as an ideal oxidant because of its abundance, atom efficiency and benign by-products.</p>
<p>
  The candidate will be involved in the following tasks: </p>
<ul>
  <li>synthesis and characterization of different peptide-based ligands containing non-canonical amino acids with good transition metal ion coordination properties <br />
    </li>
  <li>study of their metal ion coordination properties using different spectroscopic techniques (among them UV-Vis, CD, fluorescence, NMR and EPR) </li>
  <li>screen the activity of these metallopeptides in oxidation reactions using both H2O2 and O2 as green oxidants. The oxidation of alkenes, alcohols and aldehydes will be explored </li>
</ul>
<p><strong>Techniques covered in the project </strong> <br />
  Solid phase and solution peptide synthesis methodologies <br />
  Mass spectrometry <br />
  Analytical and preparative HPLC <br />
  Spectroscopic techniques: Circular dichroism, fluorescence, FT-IR, EPR, NMR, UV-Vis <br />
  GC and LC chromatography </p>
<p><strong>Minimum time (e.g. 3-6 months) </strong> <br />
  3 months, preferably 6 months </p>
<p><strong>Possible start </strong> <br />
  To be determined with the candidate </p>
<p><strong>Student requirements (e.g. GPA, courses taken, previous experience) </strong> <br />
  Preferably chemistry/biochemistry background </p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong> <br />
  Please, send by email your detailed CV, motivation letter and the contact information of two referees. </p>
<p><strong>Contact information </strong></p>
<p><img src="images/O_Iranzo.png" width="121" height="118" hspace="9" align="left" />Olga Iranzo <br />
  Institut des Sciences Moléculaires de Marseille <br />
  ISM2/BiosCiences UMR CNRS 7313, case 341 <br />
  Aix-Marseille Université, Campus Scientifique de Saint Jérôme <br />
  Avenue Escadrille Normandie-Niemen <br />
  F-13397 Marseille CEDEX 20 <br />
  Phone number: 00 33 (0)4 91 28 27 00 <br />
  Fax number: 00 33 (0)4 91 28 84 <br />
  olga.iranzo AT univ-amu.fr</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr />
<p><strong><a name="P5" id="P5"></a><h3>Improving robustness of host cells for external environments of varying and changing osmolalilites - occupied</h3></strong><br />
</p>
<p><strong>A short description of the project </strong><br />
  <img src="images/Osmol_figure.png" width="190" height="215" hspace="11" align="right" />Biological organisms offer great promises in terms of processing and manufacturing capabilities. Recent years have seen significant advances in engineering synthetic circuits designed for various applications, such as biosensing or production of useful compounds. However, synthetic circuit is not a stand-alone product and relies on successful integration with the host cell and the host cell&rsquo;s external environment. Currently this represents a major challenge. This project will be aimed towards designing a host <em>Escherichia coli</em> cell with optimal response characteristics to the changes in external osmolalities. Most of the realistic environments for synthetic biology applications are characterized with frequent, often sudden changes in external concentrations. <em>E. coli</em>, can grow in conditions ranging from 5 mOsmol/kg to ~3 Osmol/kg. In addition, it recovers rapidly from large changes, regaining growth in a matter of hours (1, 2). However, the rate of growth post recovery significantly changes (3). The contributions of different components of the osmoregulatory network (some of which might be redundant), to this modulation are not well understood, yet the changes in growth rates can negatively affect the performance of synthetic circuits. Our recent, indirect measurements indicate that the changes in growth rates are linked to the changes in cellular energetics, i.e. proton motive force (PMF). This project will aim to directly measure PMF during osmotic challenges, in singe cells and using state of the art fluorescent microscopy and optical interferometry techniques (4, 5). Time providing, the measurements, together with our current knowledge on the proteins involved in the recovery, will be used to construct a mathematical model. The model will guide our understanding of the contributions of different proteins in osmorecovery network to changes in growth rates. The outcome of the project will be further used in the future, when genetically modifying the components of the <em>E. coli</em> network to achieve a minimum osmorecovery network with characterized recovery response and minimum effects on growth rate.&nbsp;</p>
<p>
  References:<br />
  <strong>(1)</strong>Baldwin W, <em>et al</em> Growth and buoyant density of Escherichia coli at very low osmolarities (1995) Journal of Bacteriology 177:235-237. <br />
  <strong>(2)</strong> Record M, <em>et al</em> Response of <em>E.coli</em> to osmotic stress: large changes in amounts of cytoplasmic solutes and water. (1998) <em>TIBS</em> 23:143-147.<br /> 
  <strong>(3)</strong> Wood JM (2006) Osmosensing by bacteria. <em>Sci STKE</em> 2006(357):pe43. <br />
  <strong>(4)</strong> Pilizota et al A molecular brake, not a clutch, stops the <em>Rhodobacter sphaeroides</em> flagellar motor (2009) PNAS 106:11582-7 <br />
  <strong>(5)</strong> Bai et al. Experimental evidence for stochastic allostery in the bacterial switch complex (2010) Science 327:684-9 </p>
<p><strong>Minimum time (e.g. 3-6 months) </strong><br />
  the project is more suited for a 5-month period, but shorter periods will be considered</p>
<p><strong>Possible start </strong><br />
  Open.</p>
<p><strong>Student requirements (e.g. GPA, courses taken, previous experience) </strong><br />
  The student will have background in physics, microbiology, molecular biology or biochemistry. Interested applicants with background in chemistry and engineering will be considered. A strong academic record is required from interested students. </p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
  CV and a short motivational letter.</p>
<p><strong>Contact information </strong></p>
<p><img src="images/T_Pilizota.png" width="102" height="121" hspace="9" align="left" />Teuta Pilizota<br />
  Principal Investigator<br />
  Institute of Cell Biology<br />
  School of Biological Sciences<br />
  King's Buildings (Darwin Building, Room 502)<br />
  University of Edinburgh<br />
  Mayfield Road, EH9 3JR, Edinburgh, UK <br />
  teuta.pilizota AT ed.ac.uk</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr />
<p><strong><a name="P6" id="P6"></a><h3>What happens to the anomalies of super-cooled salty water? - occupied</h3></strong><br />
</p>
<p><strong>A short description of the project </strong><br />
The reason why water, cooled below freezing, shows anomalies in many thermodynamic quantities is still nowadays a matter of discussion and active research worldwide.&nbsp; In a recent work, we have demonstrated that pure liquid water when cooled below zero Celsius and under mechanical tension shows a minimum in the speed of sound [1]. This minimum represents an anomaly of the &ldquo;double metastable&rdquo; water (super- cooled and at negative pressure) and has been found both in experiments and in numerical simulations. However, in real life, water might contain salt. Therefore one might ask whether this anomaly of the speed of sound survives in an aqueous solution. <br />
  Another well-known anomaly of water cooled below zero Celsius is that water expands on freezing. At ambient pressure, its density reaches a maximum at four Celsius and decreases for lower temperatures, i.e. its volume increases. The temperature corresponding to the maximum density, Temperature of Maximum Density (TMD), can be evaluated for different pressures. The collection of all such temperatures identifies the so-called TMD line. Sastry and coworkers [2] have shown that a clear relationship exists between the TMD line and the line connecting maxima/minima of the isothermal compressibility. As a consequence, one can draw interesting conclusions on whether super-cooled water has a second liquid-liquid critical point and what will be the effect on it of the presence of salt in solution. <br />
  The first step of the project will be to become familiar with numerical simulations, learning how to simulate salty water. Water will be modeled using TIP4P/2005 interaction potential [3], known to successfully reproduce a broad variety of thermodynamic properties, whereas ion-ion and ion-water interactions will follow Ref [4]. Once the aqueous solution is ready, we will use the Molecular Dynamics package GROMACS [5] to equilibrate it at different thermodynamic conditions, e.g. temperature and pressure.&nbsp; <br />
  We will compute the speed of sound calculating the heat capacity and the isothermal compressibility. Next, we will compare the results with the ones already computed for pure water. This will allow us to quantify how the speed of sound minimum is affected by the presence of salt in water and compare with experiments of aqueous solutions. </p>
<p>To compute the TMD line, we will evaluate the temperature corresponding to the density maximum of isobaric curves (obtained by a series of simulations at constant pressure). Once more, we will be able to compare the obtained results <br />
  to a previously calculated TMD line for pure water and to experimental <br />
  measurements.&nbsp; </p>
<p>The experimental results will come from the lab of Professor Caupin at the University of Lyon. </p>
<p>[1] G.Pallares, M.El Mekki Azouzi, M.A.Gonzalez, J.L.Aragones, J.L.F. Abascal, C.Valeriani &amp; F.Caupin, &ldquo;Anomalies in bulk supercooled water at negative pressure&rdquo; </p>
<p>[2] S.Sastry, P.G.Debenedetti, F.Sciortino &amp; H.E.Stanley, Phys.Rev.E. 53, 6144 (1996).</p>
<p>[3] J.L.F.Abascal &amp; C.Vega, The Journal of Chemical Physics 123, 234505 (2005). </p>
<p>[4] K. P. Jensen &amp; W. L. Jorgensen, J. Chem. Theory Comput. 2, 1499 (2006).</p>
<p>
  [5] B.Hess, C. Kutzner, D.van der Spoel &amp; E.Lindahl, Journal of Chemical Theory and Computation 4, 435–447 (2008).</p>
<p><strong>Minimum time (e.g. 3-6 months) </strong><br />
  Open.</p>
<p><strong>Possible start </strong><br />
  Open.</p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
  CV and a short motivational letter.</p>
<p><strong>Contact information </strong><br />
  <img src="images/C_Valeriani.png" width="138" height="123" hspace="9" vspace="9" align="left" />Chantal Valeriani <br />
  Juan de la Cierva and Marie Curie Integration Grant Fellow <br />
  cvaleriani AT quim.ucm.es<br />
  Departamento de Quimica Fisica I Facultad de Ciencias Quimicas<br />
  Avenida Complutense&nbsp; <br />
  Universidad Complutense de Madrid<strong></strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr />
<p><strong><a name="P7" id="P7"></a><h3>Developing new simulation methods for the modelling of supramolecular Interactions - occupied</h3></strong><br />
</p>
<p><strong>A short description of the project</strong><br />
Reversible supramolecular interactions are now broadly used in many fields including DNA nanotechnology applications, reversible gels and reversible networks. In spite of the big expectation related to the self-assembly properties of these systems, systematic methods aimed to the modelling of reversible interactions are not yet available. This project aims to take a step toward the fulfilling of this gap. In particular, the main scope is the development of computational tools to compute the configurational costs of using reactive polymers to mediate supramolecular interactions. Some advances have been recently proposed in the field of DNA coated colloids. In particular the configurational free energy cost of hybridising diluted single stranded DNA has been recently computed using static Monte Carlo simulations (Rosenbluth sampling). This method is limited to the case of short polymers and to the low dilution limit (when polymer-polymer interactions between chains that do not hybridise can be neglected). In the present project we would release this over simplification and use configurational Biased Monte Carlo simulations to compute the density of state of functionalised particle with reactive terminals. First we will test the new methods using functionalised microparticle, for which known results are available. We will then push our modelisation in the regime of `densely-functionalised' nanoparticle and will compare our finding with other more expensive numerical procedure.</p>
<p><strong>Minimum time (e.g. 3-6 months) </strong><br />
  Open.</p>
<p><strong>Possible start </strong><br />
  Open.</p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
  CV and a short motivational letter.</p>
<p><strong>Contact information </strong></p>
<p><img src="images/B_Mognetti.png" width="94" height="125" hspace="9" align="left" />Bortolo M. Mognetti<br />
  Universite Libre de Bruxelles, Bruxelles (Belgium)<br />
  b.mognetti AT gmail.com<br />
  bm411 AT cam.ac.uk</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr />
<p><strong><a name="P8" id="P8"></a><h3>A computational genomics approach to enhancer-promoter interactions - occupied</h3></strong><br />
</p>
<p><strong>A short description of the project </strong><br />
  BACKGROUND: Expression of thousands of genes in vertebrate genomes is regulated by interactions of promoters with distally placed regulatory elements called enhancers. Individual genes differ by their ability to respond to regulation from large distances: while some respond only to enhancers located close to promoters, others can integrate inputs from dozens of enhancers placed at megabase distances from their promoter. This allows a nonlinear arrangement where enhancers are not always physically closest to the genes they regulate, creating problem of assigning the enhancers to correct gene during genome annotation and interpretation of genome-wide association studies.<br />
  THE PROBLEM: Genes regulated by multiple long-range enhancers are generally involved in the regulation of embryonic development, differentiation, and other processes that involve coordinated cell-cell communication and migration. The interactions between enhancers that act on them is not well understood. However, in recent years several large-scale projects have produced cell type-specific catalogues of active regulatory elements and gene expression levels. This data represents a currently under-utilised gold mine for studying the relationship between activity of enhancers and the expression of genes they regulate.<br />
  THE AIM: The aim of this project is to computationally analyse activities of enhancers and correlate them with expression activity of all genes in their genomic neighbourhood. As input, it will use publicly available enhancer activity and gene expression data from ENCODE and FANTOM projects. Its immediate goal will be to distinguish genes under developmental, extremely long-range regulation and to visualise the complexity of their regulatory inputs and their complex dynamics across different cells and tissues. The approaches will include analysing mutual information between the two activities across a panel of tissues and cell types, exploring novel approaches to the visualisation of this information, and an initial explorations of models that could aid in automatic and curated association of enhancers with the genes they control.</p>
<p><strong>Minimum time (e.g. 3-6 months) </strong><br />
  Open.</p>
<p><strong>Possible start </strong><br />
  Open.</p>
<p><strong>Student requirements (e.g. GPA, courses taken, previous experience) </strong><br />
  The student is expected to have a working knowledge of concepts of molecular biology or biochemistry, as well as some experience with programming (R, Python and/or Perl are preferred). Previous experience with using programming to solve biologically motivated projects is desirable but not required.</p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
  CV and a short motivational letter.</p>
<p><strong>Contact information </strong><br />
  <img src="images/B_Lenhard.png" width="184" height="120" hspace="9" vspace="9" align="left" />Boris Lenhard<br />
  Principal Investigator<br />
  Computational Regulatory Genomics<br />
  Institute of Clinical Sciences<br />
  Faculty of Medicine<br />
  Imperial College London<br />
  b.lenhard AT csc.mrc.ac.uk</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr/>

<p><strong><a name="P10" id="P8"></a><h3>Genetic code expansion and reprogramming - occupied</h3></strong><br />
</p>
<p><strong>A short description of the project</strong><br />
Genetic code expansion and reprogramming allows the site-specific incorporation of diverse unnatural amino acids into proteins produced in cells and animals. To make it possible for a cell to genetically encode for a designer amino acid as a building block of naturally translated protein, a new orthogonal pair of aminoacyl-tRNA synthetase and tRNA is required which will selectively load only the selected unnatural amino acid. Simultaneously, a new ‘blank’ codon is needed as a unique triplet presenting the new amino acid at the level of DNA/mRNA. The orthogonal archaeal pyrrolysyl-tRNA synthetase/tRNA (PylRS/PylT) pair, taken as the loading machinery, combined with the rare amber stop codon, taken as the new ‘blank’ sense codon, has proven to be a great platform for site-specific unnatural amino acid incorporation both in prokaryotic and eukaryotic cells.</p>
<p>Only recently we have demonstrated for the first time that these approaches are in principle extendable from cells to whole organisms, specifically model organism Caenorhabditis elegans (Greiss and Chin, 2011). Transgenic worms carrying orthogonal synthetically evolved PylRS/PylT pair have been shown to incorporate several unnatural amino acids, lysine derivatives, due to amber stop codon suppression. Being transparent and anatomically well studied, C. elegans is an ideal target for in vivo light-mediated intervention in biological processes using genetically encoded photo‐responsive amino acids, including photo-cross-linkers and photo‐caged amino acids. In my PhD project, I am trying to demonstrate that the unnatural amino acid incorporation can be controlled tissue-specifically/cell-specifically through manipulation of each of the components necessary for the orthogonal system to function in worms. As a consequence, I hope to be able to control the protein activity of target neuronal proteins in worm by light with high spatio-temporal precision, in order to activate/deactivate single neurons and study their roles in the neuroconnectome of this incredible little organism. </p>
<p><strong>Minimum time (e.g. 3-6 months) </strong><br />
  Open.</p>
<p><strong>Possible start </strong><br />
  Open.</p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
  CV and a short motivational letter.</p>
<p><strong>Contact information </strong><br />
<img src="images/inja.jpg" width="100" height="138" hspace="9" vspace="9" align="left" />Inja Radman<br />
  PhD Student <br />
  MRC Laboratory of Molecular Biology <br />
  University of Cambridge <br />
  ir261 AT cam.ac.uk<br /></p>

  <p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<hr/>

<p><strong><a name="P11" id="P8"></a><h3>Modifying G protein-coupled receptors for crystallographic studies - occupied</h3></strong><br />
</p>
<p><strong>A short description of the project</strong><br />
G protein-coupled receptors (GPCRs) are eukaryotic seven-α-helical transmembrane proteins which are involved in extracellular signal transduction across the cellular membrane. Being engaged in different fundamental physiological processes, GPCRs induce intracellular responses to a broad variety of external stimuli, including hormones and neurotransmitters, as well as odorants, pheromones, tastants and light. The human genome encodes at least 800 different GPCRs of which 400 are non-sensory receptors with potential medical relevance. Several research groups at Paul Scherrer Institute (PSI) gathered around Prof. Gebhard Schertler study structures, dynamics and interactions of GPCRs by biophysical, NMR, crystallographic, electron-microscopic and computational methods. The production of the receptors in the sufficient amounts and quality is a necessary prerequisite for their biophysical and structural studies, including X-ray crystallography. Because of their very low thermal stabilities when outside of biological membranes and their high conformational flexibility, GPCRs in general are notoriously difficult to crystallize. To address this issue, several strategies have been applied. These include modifications of receptor by point mutations, truncations or deletions of flexible parts (e.g. loops) as well as insertion of a soluble protein domain into a receptor molecule. 
The candidate will be involved in design, production and characterization of different GPCR constructs in order to find those which might be suitable for crystallographic studies. This research is a part of the project “Conformational dynamics of G protein-coupled receptors” led by Dr. Dmitry Veprintsev (http://www.psi.ch/lbr/conformational-dynamics-of-g-protein-coupled-receptors).
</p>
<p><strong>Techniques covered in the project </strong> <br />
PCR, molecular cloning, protein expression in mammalian cells, fluorescence microscopy, in-gel fluorescence, radioligand-binding assays, differential-scanning fluorimetry, different chromatographic methods, protein crystallization 
</p>
<p><strong>Minimum time (e.g. 3-6 months) </strong><br />
  3 months, preferably 6 months.</p>
<p><strong>Possible start </strong><br />
  Open.</p>
<p><strong>Student requirements (e.g. GPA, courses taken, previous experience) </strong><br />
Background in molecular biology or biochemistry.
</p>
<p><strong>Required documents for contact (e.g. CV, motivational letter) </strong><br />
  CV and a short motivational letter.</p>
<p><strong>Contact information </strong><br />
<img src="images/milic.jpg" width="100" height="100" hspace="9" vspace="9" align="left" />Dalibor Milić <br />
Laboratory of Biomolecular Research, OFLG/101 <br />
Paul Scherrer Institut <br />
CH-5232 Villigen PSI, Switzerland <br />
dalibor.milic@psi.ch <br /></p>

<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>



<p> <strong> </strong></p>
<p></p>
      <div id=copyright> 
        <p>&nbsp;</p>
<p>&nbsp;</p>
<hr />
<p>Copyright © 2013 Society for Out of Frame Education (EVO). All rights reserved.<br />
Web design: Martina Mijuskovic. Logo design: Lucija Silic.</p>
<p>&nbsp;</p>
</div>
    </div> 
  </div>
</div>
</body>
</html>
