<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Summer School of Science</title>
<link href="S3.css" rel="stylesheet" type="text/css" />
<script src="SpryAssets/SpryMenuBar.js" type="text/javascript"></script>
<link href="SpryAssets/SpryMenuBarHorizontal.css" rel="stylesheet" type="text/css" />
<style type="text/css">
<!--
a:link {
	color: #F93;
	text-decoration: none;
}
a:visited {
	text-decoration: none;
	color: #F93;
}
a:hover {
	text-decoration: none;
	color: #F30;
}
a:active {
	text-decoration: none;
	color: #F30;

	text-align: center;
}
a {
}
-->
</style>
<script type="text/javascript">
<!--
function MM_preloadImages() { //v3.0
  var d=document; if(d.images){ if(!d.MM_p) d.MM_p=new Array();
    var i,j=d.MM_p.length,a=MM_preloadImages.arguments; for(i=0; i<a.length; i++)
    if (a[i].indexOf("#")!=0){ d.MM_p[j]=new Image; d.MM_p[j++].src=a[i];}}
}
//-->
</script>
<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-39398660-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>

<body>
<div id="container">
<div id="header">
  <div id="hr_en">
    <p><a href="index_hr.html">hrvatski</a> / <em>english</em></p></div>
  <div id="links">
    <p> <a href="index.html">HOME</a> <a href="sponsors.html">SPONSORS</a> <a href="press.html">PRESS</a> <a href="sitemap.html">SITEMAP</a> <a href="faq.html">FAQ</a> <a href="contact.html">CONTACT</a></p></div>
</div>
<div id="banner"><a href="index.html" title="home" id="banner">banner</a></div>
  <div id="main_image"></div>
  <div id="main_content">
    <ul id="S3_menu1" class="MenuBarHorizontal">
      <li><a href="#">gallery</a>
        <ul>
          <li><a href="photos.html">photos</a></li>
          <li><a href="videos.html">videos</a></li>
          <li><a href="artwork.html">artwork</a></li>
        </ul>
    <li><a href="#">apply</a>
        <ul>
          <li><a href="process.html">application process</a></li>
          <li><a href="motivation.html">how to write a motivation letter</a></li>
          <li><a href="fees.html">application fees and scholarships</a></li>
          <li><a href="http://evounited-drustvoevo.rhcloud.com/s3/appForm/">application form</a></li>
        </ul>
<li><a href="#">people</a>
        <ul>
          <li><a href="organizers.html">organizers</a></li>
          <li><a href="mentors.html">mentors</a></li>
          <li><a href="alumni.html">alumni</a></li>
        </ul>
      </li>
      <li><a href="#">program</a>
        <ul>
          <li><a href="programS3_2015.html">S<sup>3</sup> 2015</a></li>
          <li><a href="programS3pp_2015.html">S<sup>3</sup>++ 2015</a></li>
          <li><a href="programS3_2014.html">S<sup>3</sup> 2014</a></li>
          <li><a href="programS3pp_2014.html">S<sup>3</sup>++ 2014</a></li>
          <li><a href="programS3_2013.html">S<sup>3</sup> 2013</a></li>
          <li><a href="programS3pp_2013.html">S<sup>3</sup>++ 2013</a></li>
          <li><a href="archive.html">archive</a></li>
        </ul>
      <li><a href="#">about</a>
        <ul>
          <li><a href="about_school.html">the school</a></li>
          <li><a href="history.html">history</a></li>
          <li><a href="vision.html">vision</a></li>
          <li><a href="activities.html">day-to-day activities</a></li>
          <li><a href="info.html">practical information</a></li>
        </ul>
    </ul>
    <div id="title"><img src="images/program.jpg" width="61" height="355" /></div>
    <div id="main_text">
      <h3>&nbsp;</h3>
      <h2>S<sup>3</sup> Pozega 2015 (<em>July 16<sup>th</sup> - July 24<sup>st</sup></em>): PROGRAM</h2>
     
<h3>PROJECTS</h3>

<ul>
    <li><strong><a href="#p1">Can we help House MD – Genomics in medicine</a></strong></li>
    <li><strong><a href="#p2">Biomolecular architecture: why some proteins have additional domains?</a></strong>
    <li><strong><a href="#p3">Constructing a laser video display</a></strong>
    <li><strong><a href="#p4">Speech recognition: doing some SIRIous business</a></strong>
    <li><strong><a href="#p5">Do-it-yourself bioreactor design – can we grow microalgae with everyday equipment?</a></strong></li>
        
</ul>

<h3>WORKSHOPS </h3>
Coming soon...
<h3>LECTURES</h3>
Coming soon...
<hr />

<div id=projects> 
      <h3>PROJECTS</h3>
      <h3><strong><a name="p1" id="p1"></a> Can we help House MD – Genomics in medicine </strong></h3>
      <p><img src="images/2015/dunja_project_photo.jpg" width="250" height="200" hspace="9" align="right" />Would you buy the same clothes as your grandmother? Probably not. But why is it that when you get sick, you’re likely to receive just about the same treatment and drug choices? And so will everyone else. That’s because even the world’s best scientists and doctors have limited knowledge of how different people respond to different treatments. The result is a "one size fits all" approach to medicine. That’s now changing. Personalized medicine is moving us closer to more precise, predictable and powerful medicine—customized for the individual patient. Our growing understanding of genetics is allowing us to provide better diagnoses, safer drug prescriptions, and more effective treatments of diseases and conditions that have affected us throughout history. Today, scientists and doctors are learning how to tailor health care to a person’s unique genetic makeup. By developing personalized medicine we hope to avoid many side effects of inappropriate treatments and shorten the overall duration of treatment. </p>
In this project every participant will get a “patient” – medical history of a patient for whom the diagnosis could not be established buy using standard medical procedures, and a genotype – complete DNA sequence derived from that person. By combining this information we will try to get better predictions of diseases our patients are suffering from. In order to be able to do this, participants will learn how to deal with genomic data, search the databases for information about diseases and perform a bit of R coding.
<p><br />
      </p>
      <p><strong><img src="images/2015/Dunja.png" height="160" hspace="9" vspace="14" align="left" />Dunja Vučenović</strong><br />
      University of Zagreb, Croatia<br /></p>
        
        <p>Dunja Vucenovic is a Master student of Molecular biology at Faculty of Science in Zagreb. During her studies she got interested in different OMICs approaches. Her S3 story began while she was still in the elementary school and since then she was a participant for 4 years in a row. Last year, Dunja joined the Society for Out of Frame Education and she has organized several workshops for the Science Picnic festival. In her free time she enjoys cooking and experimenting with different cuisines, travelling and wandering in nature. </p>
      <p><br />
        <br />
      </p>
<h3><a name="p2" id="p2"></a>Biomolecular architecture: why some proteins have additional domains?</h3>
<p><img src="images/2015/nikolina_project.png" height="200" hspace="9" vspace="9" align="right" />Genetic information is often said to be kept in form of DNA molecules. More specifically, every DNA molecule is built out of nucleotides and the important information arises from their sequence. In every cell, this sequence is transcribed in a form of mRNA molecule, which is further used for protein synthesis enabled by the activity of ribosomes. Hence, genetic information given by nucleotide sequence of mRNA molecules is translated into protein amino acid sequence. tRNA molecules also participate in a translation process as adaptor molecules that bring the corresponding amino acids to ribosomes. Faithful transmission of genetic information is ensured through the activity of enzymes that attach amino acids to cognate tRNA molecules. Some of these enzymes are found to be involved in a formation of human diseases (eg. neuronal pathologies), making a detailed investigation of these proteins of exceptional importance.</p>
<p>
  In this project, one of the enzymes catalyzing covalent bond formation between the amino acid and tRNA molecule is going to be further explored. Experimental part of the project is going to begin by growing genetically modified bacteria that produce large quantites of the abovementioned protein. This protein is going to be purified from all other bacterial proteins and its affinity to tRNA is going to be determined. Experimental findings are further going to be supported by computational analyses by which we are going to simulate the movement of our protein. In other words, the molecular interactions from the experimental part are going to be investigated using the molecular dynamics data at the atomic level.  </p>
<p>&nbsp;</p>
<p><strong><img src="images/2015/nikolina_slika.jpg"  height="150" hspace="9" vspace="9" border="0" align="left" />Nikolina Sostaric</strong><br />

  University of Zagreb, Croatia<br />
</p>
<p>Nikolina is a 5<sup>th</sup> year student of Chemistry at Faculty of Science in Zagreb. She participated in Summer School of Science 2013 (S<sup>3</sup>++ camp) as a swapshop leader. Nikolina also volunteered in the student project Magic in Science and currently works as a student mentor in the Fifth Gymnasium in Zagreb. She is interested in enzyme kinetics, molecular simulations and statistical analysis of biological data. In her free time, Nikolina enjoys reading, cooking and painting. </p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<h3><strong><a name="p3" id="p3"></a>Constructing a laser video display</strong></h3>
<p><img src="images/2015/renan_project_photo.jpg"  height="205" hspace="9" align="right" />In principle, laser displays are simple: they consist of a laser which fires onto two mirrors; one mirror vibrates on a horizontal axis, and the other vibrates on a vertical axis. This makes the laser beam scan across a screen. By turning the laser on and off at the right times, you can draw any picture you like. Take red, green, and blue lasers, and you have your own color TV! In practice, things are never that simple. How do you build and control the mirrors, which must vibrate at extremely high rates? How do you synchronize between the mirrors and the lasers? How do you actually combine colors? Answer these questions, and you'll be able to manufacture tiny yet powerful handheld projectors, a growing and important field in the display industry. </p>
In this project, we will build a basic laser video display using only commonly found household items such as laser pointers and speakers. While not quite supporting HD resolution, the display will be able to show primitive shapes and pixels, and will be controlled through a computer and an Arduino microcontroller (a small external computer than can easily be programmed to interact with hardware). Building such a device requires diving into basic optics and fundamentals of oscillations, resonance and acoustics. Signal processing will also come into play. We will have to overcome a number of structural challenges; glue, nails, and soldering is to be expected. In the end, we will have combined theory and practice to build a machine that demonstrates important principles in display technology. 
<p> <br />
</p>
<p><img src="images/2015/renan_gross_photo.png"  height="144" hspace="9" vspace="7" align="left" /><strong>Renan Gross</strong><br />
  Technion – Israel Institute of Technology, Israel</p>
<p>Renan Gross is currently finishing his undergraduate degree in mathematics and physics at the Technion - Israel Institute of Technology. He is interested in all fields of science, from mathematics and computers to physics, informatics and biology. During his spare time he writes about science, politics and life, plays the piano, and occasionally performs poetry. Renan mentored thrice in the Summer School of Science: as a swapshop leader in 2011, and as a project leader in 2012 and 2013.</p>
<br />
<p>&nbsp;</p>

<h3><strong><a name="p4" id="p4"></a>Speech recognition: doing some SIRIous business</strong></h3>
<p>
<img src="images/2015/jessa_project_photo.jpg"  height="216" hspace="9" align="right" />"Hey Siri, find me some summer schools in Europe!" might have led you here. Your mobile personal assistant who understands your voice makes interacting with a machine very natural. This is how it should be: computers are meant to make certain tasks easier for humans. In order to to this, the machines must not only be able to help with the task at hand, but communicating with it should also be easy The first computers did not have keyboards nor screens, input had to be provided by connecting wires in a certain way. But we have come a long way since then. Now, apart from keyboard, screen and mouse, we can use touch, 3D camera's and even voice control to interact with our computers. Speech recognition still feels like science fiction, but is really making its entrance into our world. We know it from Siri and Google Now, our mobile personal assistants. But it is also used to (partially) replace call centers, for automatic subtitling and automatic translation. Automatic translation makes a conversation possible between people who do not speak the same language: The computer interprets spoken language, translates it and says it in the target language. </p>
<p>In this project we will build our own personal assistant: A computer program to which we can give spoken commands. In order to achieve this, the program needs to map our voice to phones, words and even sentences. The program will use machine learning which means it will learn from examples, in this case from voice samples of the students themselves. We will conduct experiments to determine which techniques work best and which phones/words/sentences are easy or difficult to recognize for a computer. </p>
<p><br />
</p>
<p> <br />
  <img src="images/2015/jessa.jpg" alt="" height="130" hspace="9" vspace="7" align="left" /><strong>Jessa Bekker</strong><br />
  KU Leuven, Belgium </p>
<p>Jessa graduated Computer Science at the KU Leuven in Belgium. She is currently a PhD student and teaching assistant at the same institution. Her research is about machine learning, in particular how to make fast predictions. Next to teaching engineering students, she also mentors young programmers by being a CoderDojo coach. In her free time, Jessa likes to spend time with her friends, travel and play the piano. </p>
<p>&nbsp;</p>
<p></p>
<p>&nbsp;</p>


<h3><strong><a name="p5" id="p5"></a>Do-it-yourself bioreactor design – can we grow microalgae with everyday equipment?</strong></h3>
<p>
<img src="images/2015/MBuffing_project.jpg" height="180" hspace="9" align="right" />Many of the products we use every day are made with the help of microorganisms. Think about bread, beer, insulin, biofuels, and much, much more. To make the production as efficient as possible, we have to think about how exactly we need to grow these organisms to make them grow as fast as possible. That is exactly what we do in bioreactor design: we try to make the microorganisms as comfortable as possible so they will grow fast. But not all microorganisms need the same kind of environment. Therefore, we need to test what makes them happy. Do they need a lot of sugar? What pH do they need? Do they need oxygen and how much? What kind of reactor fits best with their needs? </p>

<p>In this project we try to answer these questions for fast growing microalgae. Microalgae are unicellular organisms that can grow on light using photosynthesis. Over the last years, there is a lot of attention on microalgae, because they can be used to produce food supplements (eg. carotene) for example and biofuels. First, we will use everything we know about algae to design a bioreactor that suits algae’s needs. After that we will build our own bioreactor and test how we can optimize the production by changing different parameters. At the end of the project we will propose how microalgae can be produced as efficiently as possible. </p>
<p><br />
</p>
<p> <br />
  <img src="images/2015/MBuffing_pic.JPG" alt="" height="160" hspace="9" vspace="7" align="left" /><strong>Marieke Francisca Buffing</strong><br />
  ETH Zürich, Switzerland </p>
<p>Marieke studied biotechnology at Wageningen University (in the Netherlands), where she also did her Master’s in process engineering and molecular biotechnology. During her studies she got more and more interested in mechanisms that cells use to adapt to their environment. Now, during her PhD at ETH Zurich, Marieke is trying to find out how bacterial species use different protein-metabolite interactions to make fast decisions when their environment changes. When she is not in the lab, Marieke is mostly in museums or running around in the forest near the campus. </p>
<p>&nbsp;</p>
<p></p>
</div>

<p></p>
      <div id="copyright">
        <p>&nbsp;</p>
        <p>&nbsp;</p>
        <hr />
        <p>Copyright © 2014 Society for Out of Frame Education. All Rights Reserved.<br />
          Webdesign: Martina Mijuskovic. General text: Branimir Lukic and Martina Mijuskovic.  Croatian translation: Matilda Males.</p>
        <p>&nbsp;</p>
      </div>
    </div>
</div>
<script type="text/javascript">
<!--
var MenuBar1 = new Spry.Widget.MenuBar("S3_menu1", {imgDown:"SpryAssets/SpryMenuBarDownHover.gif", imgRight:"SpryAssets/SpryMenuBarRightHover.gif"});
//-->
</script>
</body>
</html>
