<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Summer School of Science</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/sss.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="http://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    <link href="http://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    
    <script src="js/jquery-1.6.1.min.js"></script>
	<script src="js/bootstrap.min.js"></script>
	<script src="js/init.js"></script>

    <!-- Google Analytics -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-64967324-1', 'auto');
      ga('send', 'pageview');

    </script>

</head>

<body id="page-top">

    <!-- Navigation -->
    <nav class="navbar navbar-custom navbar-fixed-top top-nav-collapse" role="navigation">
        <div class="container">
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                    <i class="fa fa-bars"></i>
                </button>
                <a class="navbar-brand page-scroll" href="index.html">
                <span class="light" class="light" style="font-family: Arial;">S<sup style="font-family: Arial;">3</sup>/S<sup style="font-family: Arial;">3</sup>++</span>
                </a>
            </div>

            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse navbar-right navbar-main-collapse">
                <ul class="nav navbar-nav">
                    <!-- Hidden li included to remove active class from about link when scrolled up past about section -->
                    <li class="hidden">
                        <a href="#page-top"></a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="index.html">About</a>
                    </li>
                    <li class="dropdown">
                        <a href="#" data-toggle="dropdown" class="dropdown-toggle nav">Program <b class="caret"></b></a>
                        <ul class="dropdown-menu">
                            <li><a href="s3_2017.html">S<sup>3</sup> 2017</a></li>
                            <li><a href="s3pp_2017.html">S<sup>3</sup>++ 2017</a></li>
                            <li><a href="archive.html">Archive</a></li>
                            <li><a href="daytoday.html">Day-to-day activities</a></li>
                            <li><a href="index.html#practical">Practical info</a></li>
                        </ul>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="newsfeed.html">News</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="people.html">People</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="apply.html">Apply</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="sponsors.html">Sponsors</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="press.html">Press</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="faq.html">FAQ</a>
                    </li>
                    <li>
                        <a class="page-scroll nav" href="index.html#contact">Contact</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

    
    <!-- About Section11 -->
    <section id="about" class="content-section text-center">
        <div class="break-section2">
            <div class="container">
            	<div class="col-lg-8 col-lg-offset-2">
                    <h2>S<sup>3</sup> 2005</h2>
                    <p > Višnjan, July 24<sup>th</sup> - August 2<sup>nd</sup></p>
            
                    <a href="#projects" class="page-scroll btn btn-default btn-lg">Projects</a>
                    <a href="#lectures" class="page-scroll btn btn-default btn-lg">Lectures</a>

                    
                </div>
            </div>
        </div>    
    </section>






    <section id="projects" class="content-section text-center">
        <div class="break-section2">
            <div class="container">
                <div class="col-lg-8 col-lg-offset-2">
                    <h2>Projects</h2>
                    <div class="lineb"></div>
                    
                    <p class="media">

                        <h3><strong><a name="p1" id="p1"></a>brownian motion studied by an optical trap</strong></h3>
                        <p class="project"><img src="img/2005/pr01.jpg" width="200" height="200" hspace="9" align="right" />Optical traps are commonly used in biophysics and soft-matter physics to study motion and mechanical properties of nanoscale objects. The goal of the group is to build a functional optical trap that uses highly focused light beam to capture and move spheres of one micrometer in diameter. By recording its motion, we will explore properties of free and confined Brownian motion. If time permits, we will briefly study cell motility.  
                        </p>

                        <p class="project">This project offers a unique mix of theory and experiment where participants will learn principles of instrument design and characterization, as well as essentials of statistical physics. They will also work on data analysis and numerical simulations.</p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2005/leader01.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Branimir Lukic</strong><br />
                               Swiss Federal Institute of Technology (EPFL), Lausanne, Switzerland<br/><br/>

                        Branimir is a doctoral student in physics at the Institute of Physics of Complex Matter. He mainly works with atomic force microscopy and optical traps to study nanomechanics of carbon nanotubes and biological structures. He participates in the S3 since its beginning in 2001. 
                            </p>
                        </div>
                    </div>

                    <div class="line2"></div>

                    <h3><strong><a name="p1" id="p1"></a>investigating molecular evolution by biochemical analysis of fish proteins</strong></h3>
                        <p class="project"><img src="img/2005/pr02.jpg" width="200" height="180" hspace="9" align="right" />Each part of an organism has its own defined composition of proteins, which is required for its specific function. For instance, muscular tissue expresses a variety of proteins required to perform physical movement, whereas skin tissues express other proteins, which build a barrier and protection of the organism against various environmental stress. In this project, students will analyze muscle tissue from fish and learn how to visualize and to purify a specific set of proteins. Results obtained from different fish samples will be compared to the existing evolutionary trees. The relationship between genetic information, protein expression and cellular function will be extended in individual theoretical and bioinformatical projects. Experimental part will be accompanied with lectures and discussions and scientific communication skills are introduced through designing posters and presentations.

                        </p>

                        
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2005/leader02.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Isabel Kurth</strong><br />
                              Swiss Institute for Experimental Cancer Research (ISREC), Lausanne, Switzerland<br/><br/>

                           Isabel is a doctoral student at ISREC, Lausanne, Switzerland. Her research interest focuses on how eukaryotic telomeres are replicated and maintained by telomerase reverse transcriptase (telomerase) and by other telomere binding proteins. As part of the project she is biochemically characterizing human telomerase. 
                            </p>
                        </div>
                    </div>

                    <div class="line2"></div>

                    <h3><strong><a name="p1" id="p1"></a>Contructing a wireless sensor network</strong></h3>
                        <p class="project"><img src="img/2005/pr03.jpg" width="220" height="200" hspace="9" align="right" />Pebble-sized, low-power, embedded computers self-organized in loose networks. This concept that not so long ago belonged in science fiction magazines, came to life with the advent of cheap, mass-producible embedded computers and sensors. With industrial, scientific, agricultural and even military applications, wireless sensor networks, sometimes referred to as 'smart dust', are a fascinating solution to many previously insurmountable everyday problems.  
                        </p>

                        <p class="project">The project will give the students a flash-overview of wireless communications, and will then focus on wireless sensor networks. Routing, reliable delivery, and evolving network topologies will be discussed in the context of specific applications. The students will be asked to design, implement and test a simple routing algorithm that will execute on a Telos-series wireless sensor network.

                        </p>

                       
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2005/leader03.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Ivan Krstic</strong><br />
                               Faculty of Arts and Sciences, Harvard University, USA <br/><br/>

                           Ivan Krstic is a student of computer science at the Faculty of Arts and Sciences, Harvard University. Currently, he is the Director of Research at the Medical Informatics Laboratory at Zagreb Children's Hospital in Croatia. His interests include distributed systems and algorithms, multi-agent systems, applied and theoretical mathematics, cryptography, and large-scale web applications. 
                            </p>
                        </div>
                    </div>

                    <div class="line2"></div>

                    <h3><strong><a name="p1" id="p1"></a>deducting protein 3D structure from its amino acid sequence</strong></h3>
                        <p class="project"><img src="img/2005/pr04.jpg" width="210" height="250" hspace="9" align="right" />Proteins are macromolecular building blocks of all living organisms, composed of various numbers of only 20 amino acids. Protein sequence, i.e. the order of amino acids in its chain is unique for every protein, as well as their three-dimensional arrangement. Protein 3D structure is necessary for deeper understanding of its function- however, it is a very challenging task to obtain it experimentally. A major goal of structural bioinformatics is to predict the 3D structure from the protein sequence. One method that can be applied is homology modeling, a procedure that develops protein 3D structure based on its sequence and the known structures of similar (homologous) proteins.
                        </p>

                        <p class="project">The students will each be given a different protein sequence to study. They will learn how to extract the wealth of information stored in a protein sequence, like establishing evolutionary relationship between proteins and getting hints on their structure and function in the cell. The tertiary structure of a protein will be obtained by homology modeling through a series of computational steps.  </p>

                                        
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2005/leader04.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Anita Krisko</strong><br />
                               Rudjer Boskovic Institute, Zagreb, Croatia<br/><br/>

                          Anita is a doctoral student in molecular biophysics at Rudjer Boskovic Institute, Zagreb, Croatia. Her research is mainly focused on structure-function correlation in human low density lipoproteins. Her interests also include structural biology and bioinformatics. 
                            </p>
                        </div>
                    </div>

                    <div class="line2"></div>

                    <h3><strong><a name="p1" id="p1"></a>studying imapct craters in loose media</strong></h3>
                        <p class="project"><img src="img/2005/pr05.jpg" width="220" height="200" hspace="9" align="right" />Although geophysicists have long been interested in impact craters, several properties of these craters were not experimentally studied in a lab until recently. This group experiments with low-speed impact craters of different balls in sand and aims to derive scaling laws describing the behavior without full theoretical understanding of the processes involved. Furthermore, a simple numerical impact simulation can be developed and compared to real data. 
                        </p>

                        <p class="project">The project involves experimental measurements and analysis of data with acquisition errors. Participants will also see the principle of scaling, optionally learn about numerical simulations and maybe even a bit of electronics.

                        </p>
                    
                    <div class="media">
                        <div class="media-left">
                            <a href="">
                                <img class="media-object" src="img/2005/leader05.jpg" width="135" hspace="9" vspace="14" align="left" />
                            </a>
                        </div>
                        <div class="media-body">
                            <p class="project"><strong>Wolfgang Wieser</strong><br />
                               University of Munich (LMU), Germany<br/><br/>

                          Wolfgang is a final year physics student at the University of Munich, Germany. He is working as a designer of digital electronics at the LMU and will start his diploma thesis this year in the field of quantum optics. His further interests include realistic image synthesis, numerical simulations and some photography and biology. 
                            </p>
                        </div>
                    </div>

                    <div class="line2"></div>

                    
                    </p>
                </div>
            </div> 
        </div>
    </section>



                      <section id="lectures" class="content-section text-center">
        <div class="break-section">
            <div class="container">
                <div class="col-lg-8 col-lg-offset-2">
                    
                    
                    <div id="swapshops"


                        <div id="swapshop">
                    
                    <h3>Lectures </h3>

                   <p class="main1"> We plan several lectures during the school. Tentative list of speakers is given below.</p>

                   <p class="main2">
                        <strong>Goran Zgrablic</strong> (Swiss Federal Institute of Technology, Lausanne, Switzerland): <i>Biomimetic materials - Nature is teaching us how to design new molecules</i>
                    </p>

          

                    <p class="main2">
                        <strong> Camilo Guzman </strong> (Swiss Federal Institute of Technology, Lausanne, Switzerland): <i>Keeping the mechanical integrity of the cells: A tough work done by a not so tough element</i>
                    </p>

                    <p class="main2">
                        <strong> Branimir Lukic</strong> (Swiss Federal Institute of Technology, Lausanne, Switzerland): <i>Manipulation in the microworld: moving objects by light</i>
                    </p>

                    <p class="main2">
                        <strong>Bojan Zagrovic </strong> (Swiss Federal Institute of Technology, Zurich, Switzerland): <i>Protein folding problem: biology meets chemistry and physics</i>
                    </p>

                     <p class="main2">
                        <strong>Martina Mijuskovic </strong> (Swiss Federal Institute of Technology, Zurich, Switzerland): <i>How is evolution written in the DNA sequences?</i>
                    </p>
                
                <p class="main2">
                        <strong>Nikola Biliskov </strong> (Rudjer Boskovic Institute, Zagreb, Croatia): <i>Liquid state: between ordered structure and complete chaos</i>
                    </p>

                    <p class="main2">
                        <strong>Iva Tatic </strong> (Pliva Research Institute, Zagreb, Croatia): <i>From computer to sore throat in few "easy" steps</i>
                    </p>

                    <p class="main2">
                        <strong>Irena Jevtov </strong> (University of Zurich, Switzerland): <i>Signaling network that reveals why is an elephant bigger than a mouse</i>
                    </p>
                        
                       
                    
                        </p>

                </div>
                
            </div>     
        </div>

        </section>


                       
                                    
                    </div>
                </div>
            </div>
        </div>
    </section>



    <!-- Footer -->
    <footer>
    <hr> 
        <div class="container text-center">
            <p><font size="2">Copyright &copy;  Society for Out-of-Frame Education. All Rights Reserved.</font></p>
                <p><font size="2">Webdesign: Sebastijan Dumančić, Igor Jukić, Lucija Šilić, Jelena Tica. <br> General text: Sebastijan Dumančić, Petra Krznar, Branimir Lukić, Martina Mijušković, Nikolina Šoštarić </font></p>
        </div>
    </footer>

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Plugin JavaScript -->
    <script src="js/jquery.easing.min.js"></script>

    <!-- Custom Theme JavaScript -->
    <script src="js/single.sss.js"></script>

    

</body>

</html>

